code/utils/variance-per-voxel.py

```diff
@@ -0,0 +1,171 @@
+# If you want to show figures inside iPython notebook
+# %matplotlib inline
+
+# Importing modules and tools
+import numpy as np 
+import matplotlib.pyplot as plt 
+import nibabel as nib
+
+# import sys
+# Appending directoy to sys.path
+#sys.path.append('/Users/ying/Stat159/data/sub001/BOLD/task001_run001')
+# fname = bold_dico_dico7Tad2grpbold7Tad.nii
+
+def load_image(fname):
+	"""
+	Loads the .nii file 
+
+	Parameters
+	----------
+	fname : String
+	The path to the .nii file that we wish to load and get the data of
+
+	Returns
+	-------
+	data : a 4-D array
+	"""
+	img = nib.load(fname)
+	data = img.get_data() # data.shape yields (132, 175,  48, 451)
```

Comment:
I would return im.get_data() immediately without storing it in a variable, since that variable is only being referenced once and get_data() self documents what the value is